getting-started/README.adoc

```diff
@@ -85,6 +85,8 @@ The new Kafka instance is listed in the instances table. When the instance *Stat
 [.screencapture]
 .Kafka instance options menu
 image::sak-kafka-instance-options.png[Image of Kafka instance options menu]
+
+NOTE: Only the user who creates a Kafka instance can edit or delete the instance, access the associated service account and topics, and connect to the instance. All other users can access and connect to only their own instances.
```

Comment:
Thanks for capturing this! Could we:

- Use second person (you) to match the rest of the text (not sure if that would derail re-use)?
- Drop the second sentence, since it's implied by the first?

Reply:
@jciroli-rh ack. I wanted to do the first but thought I'd keep it general enough so that Paul and John could copy-paste and have it be a general note wherever needed. But what do you think about @jbyrne-redhat 's rewrite below? Maybe continue the convo there.

Reply:
What Jen said, although I'm struggling to re-phrase it for second person in my mind. (Opinion: As written doesn't give me heartburn, because I feel like a Note already somewhat breaks the tone; it's naturally different because it's a Note.)

Reply:
ack. (back to not being able to resolve)